lib/driver/pg.js

```diff
@@ -290,8 +290,18 @@ var PgDriver = Base.extend({
         } else {
           sql = util.format('ALTER TABLE "%s" ALTER COLUMN "%s" DROP DEFAULT', tableName, columnName);
         }
+        return this.runSql(sql).then(
+          setType.bind(this)
```

Comment:
one thing left, you need to return this, not just calling it. The promise wont resolve the result if you don't return it within the promise itself